src/app/index.html

```diff
@@ -14,167 +14,77 @@
   <script src="lib/jasmine-3.2.1/jasmine-html.js"></script>
   <script src="lib/jasmine-3.2.1/boot.js"></script>
 
-  <!-- Theme used for syntax highlighting of code -->
-  <link rel="stylesheet" href="lib/css/zenburn.css">
-  <link rel="stylesheet" href="css/main.css">
+		<!-- Theme used for syntax highlighting of code -->
+		<link rel="stylesheet" href="lib/css/zenburn.css">
+		<link rel="stylesheet" href="css/main.css">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<link rel="stylesheet" href="css/swenCustom.css">
+	  
+		<!-- jQuery library -->
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+		<script src="https://cdn.jsdelivr.net/npm/ua-parser-js@0/dist/ua-parser.min.js"></script>
 
-  <script src="https://cdn.jsdelivr.net/npm/ua-parser-js@0/dist/ua-parser.min.js"></script>
-
-  <script src="js/cycaller.js"></script>
-  <script src="js/slides/session.js"></script>
-  <script src="js/slides/slidesModel.js"></script>
-  <script src="spec/cytoscape_spec.js"></script>
-  <!-- Printing and PDF exports -->
-  <script>
-    var link = document.createElement('link');
-    link.rel = 'stylesheet';
-    link.type = 'text/css';
-    link.href = window.location.search.match(/print-pdf/gi) ? 'css/print/pdf.css' : 'css/print/paper.css';
-    document.getElementsByTagName('head')[0].appendChild(link);
-  </script>
-</head>
-  <body>
-    <div id="log-div">
-      <div id="log-button-container">
-        <input type="button" id="log-button" onClick="toggleLog()" value="LOG"/>
-        <!-- <input type="button" id="jiraBtn" onClick="submitReport()" value="Jira"/>
-        <input type="button" id="jiraBtn2" onClick="attach()" value="Attach"/> -->
-
-      </div>
-      <div id="log-container" style="display:none">
```

Comment:
Steve, How do you plan to expose runtime errors on the UI? do you plan to have an error container to display caught errors, or use buildslide html to render the errors?

Reply:
I think that's outside the scope of this. But that being said, it can be as simplying as using some bootstrap alert/toaster in the application.